### DISCUSSION TITLE
what about spectrum community archive?

### DISCUSSION THREAD

**Discussion by akaguny** [mobxjs/mobx-state-tree#1678] [Q&A] [answered]
subj

**Comment by EmilTholin** [comment, marked as answer]
Hi @akaguny!

The current `mobx-state-tree has a new home - This community is no longer on Spectrum.` view on Spectrum is sadly the only available option for an archived community. Existing posts are still reachable through direct links, but I would guess they have done it this way since Spectrum has been extremely unreliable lately and they want to cut down on traffic.
